python/mlx/utils.py

```diff
@@ -1,5 +1,4 @@
 # Copyright © 2023 Apple Inc.
-
```

Comment:
Probably removed unnecessarily when removing the original Python implementation of the context manager.